async-openai/src/client.rs

```diff
@@ -29,15 +29,23 @@ pub const API_BASE: &str = "https://api.openai.com/v1";
 /// Name for organization header
 pub const ORGANIZATION_HEADER: &str = "OpenAI-Organization";
 
-impl Client {
-    /// Create client with default [API_BASE] url and default API key from OPENAI_API_KEY env var
-    pub fn new() -> Self {
+impl Default for Client {
+    // Create client with default [API_BASE] url and default API key from OPENAI_API_KEY env var
+    fn default() -> Self {
         Self {
             api_base: API_BASE.to_string(),
             api_key: std::env::var("OPENAI_API_KEY").unwrap_or_else(|_| "".to_string()),
-            ..Default::default()
+            org_id: Default::default(),
+            backoff: Default::default(),
         }
     }
+}
+
+impl Client {
+    /// Create client with default parameters
```

Comment:
```suggestion
    /// Create client with default [API_BASE] url and default API key from OPENAI_API_KEY env var
```
Preserve comment for doc.rs?